docs/user/getting-started.md

```diff
@@ -0,0 +1,260 @@
+# Getting Started
+
+The following is an opinionated way of getting started with Flight Control on a local Kind cluster. Please refer to [Installing the Flight Control Service](installing-service.md) and [Installing the Flight Control CLI](installing-cli.md) for the full documentation including other installation options.
+
+## Deploying a Local Kind Cluster
+
+Install the following prerequisites by following the respective documentation:
+
+* `kind` version v1.29 or later ([installation guide](https://kind.sigs.k8s.io/docs/user/quick-start/))
```

Comment:
The kind doc indicates v0.23.0 is the latest, the pre-built images have v1.29.

Reply:
Yeah maybe drop the version for latest? Maintaining versions never works out well.

Reply:
[x] done